SignalServiceKit/protobuf/OWSSignalServiceProtos.proto

```diff
@@ -143,15 +146,14 @@ message SyncMessage {
     optional uint64 timestamp = 2;
   }
 
-  optional Sent     sent       = 1;
-  optional Contacts contacts   = 2;
-  optional Groups   groups     = 3;
-  optional Request  request    = 4;
-  repeated Read     read       = 5;
-  optional Blocked  blocked    = 6;
-  optional Verified verified   = 7;
-  optional bytes    padding    = 8;
-  optional bytes    profileKey = 9;
```

Comment:
Most of this block is reformatting to align with Android's protos, but I did also remove the profileKey field from SyncMessages.